python/lsst/dax/apdb/apdb.py

```diff
@@ -909,9 +895,7 @@ def quoteId(columnName: str) -> str:
             return columnName
 
         if conn.engine.name == "oracle":
-            return self._storeObjectsAfwOracle(objects, conn, table,
-                                               schema_table_name, replace,
-                                               extra_columns)
+            raise RuntimeError("Oracle backend is not supported")
```

Comment:
This method disappeared in later commit, I think there is no point trying to improve it. I don't think we'll ever try to run with Oracle again, and I'm sure it will break in unexpected ways if we try.